src/Docker/Configuration/State.php

```diff
@@ -26,7 +26,14 @@ public function getConfigTreeBuilder()
                                     ->end()
                                 ->end()
                             ->end()
-
+                            ->arrayNode(StateFile::NAMESPACE_FILES)
+                                ->prototype('array')
+                                    ->children()
+                                        ->variableNode('tags')->isRequired()->end()
```

Comment:
this doesn't have to be variable node, we know what it is

Reply:
right,  for some reason it was messing with me as an array node :-).  but you're right, it will always be source tag style, I'll update it